README.md

```diff
@@ -67,15 +67,33 @@ make sky130hd_temp
 Please contact mehdi@umich.edu if you have any questions.
 
 
-#Generators
+# Generators
+
+**[temp-sense-gen:-](https://github.com/idea-fasoc/OpenFASOC/tree/main/openfasoc/generators/temp-sense-gen)** A fully automated SoC generator that uses an all-digital temperature sensor architecture, that relies on a new subthreshold oscillator (achieved using the auxiliary cell “Header Cell“) for realizing synthesizable thermal sensors.
```

Comment:
Can you add a link to a readme on each of the generators instead of having them on the main page?

Reply:
I couldn't find the readme file for any of the generators in their main pages or other folders.

Reply:
you need to create one for each and put the info there